include/protectedfds.h

```diff
@@ -22,31 +22,72 @@
 #ifndef DMTCPPROTECTEDFDS_H
 #define DMTCPPROTECTEDFDS_H
 
+#include <stdint.h>
+#include <stdlib.h>
+
+#define DEFAULT_PROTECTED_FD_BASE   820
+
+uint32_t protectedFdBase();
+
+inline uint32_t protectedFdBase()
+{
+  static uint32_t base = DEFAULT_PROTECTED_FD_BASE;
```

Comment:
Let's set `DEFAULT_PROTECTGED_BASE` to either `(uint)-1` or `1<<30`, so as to create an immediate error if anyone tries to use this.  This might be more future-proof than setting it to the old `820`.